testutil/keeper/supplier.go

```diff
@@ -22,7 +22,11 @@ import (
 	"github.com/pokt-network/poktroll/x/supplier/types"
 )
 
-func SupplierKeeper(t testing.TB, sessionByAppAddr supplier.SessionsByAppAddress) (*keeper.Keeper, sdk.Context) {
+// SupplierKeeper returns a mocked supplier keeper and context for testing.
```

Comment:
(same as https://github.com/pokt-network/poktroll/pull/319#discussion_r1453313810). It's more like it returns a keeper with mocked dependencies.